### ISSUE TITLE
Load by demand

### ISSUE THREAD

**Issue by Josema** [crypto-browserify/crypto-browserify#170]
Is there any options to browserify less code. Using just AES encryption I got a 300kb min file. Maybe using rollup or loading sub modules `require('crypto/aes')`.

**Comment by calvinmetcalf** [comment]
all of the module are available to require individually so you could require `browserify-cipher` directly

**Comment by coltongit** [comment]
Agreed, better to just require `browserify-cipher` and get just the code for that that way. Could this perhaps be closed by any chance?
